_rules/first-focusable-element-is-link-to-non-repeated-content-8a213c.md

```diff
@@ -0,0 +1,658 @@
+---
+id: 8a213c
+name: First focusable element is link to non-repeated content
+rule_type: atomic
+description: |
+  This rule checks that the first focusable element is a link to non-repeated content in the page
+accessibility_requirements:
+  wcag-technique:G1: # Adding a link at the top of each page that goes directly to the main content area
```

Comment:
This rule does not map to WCAG directly. We had this issue in #1496. Have no issues with reviewing but the TF may not publish it. Maybe it's worth tweaking it so that it could be part of a composite rule?